server/lib/src/server/mod.rs

```diff
@@ -901,13 +904,27 @@ pub trait QueryServerTransaction<'a> {
             })
     }
 
+    fn get_apprs_set(&mut self) -> Result<Vec<Arc<EntrySealedCommitted>>, OperationError> {
```

Comment:
Why do you have this twice? with get_apprs and get_application_rs_set?

Reply:
A mistake, fixed.